### ISSUE TITLE
Add filtering capabilities

### ISSUE THREAD

**Issue by samvv** [TypeForce/tsdoc#32]
Allow entries to be filtered based on several criteria, the most important being which file they originated from (is it a project source file? a node module?).

**Comment by BurtHarris** [comment]
I'm not clear on what you mean here.   Are these filters something that might be implemented in an existing gulp plugin?   From my point of view, in TypeScript/nodejs, a source file is the same thing as a module.

**Comment by samvv** [comment]
That's true; actually in most cases that will do. I was thinking of custom filters for advanced use cases, like filtering `@internal` and properties beginning with a `_`.
